python/ray/services.py

```diff
@@ -367,6 +371,51 @@ def check_version_info(redis_client):
             print(error_message)
 
 
+def start_credis(node_ip_address,
+                 port=None,
+                 redirect_output=False,
+                 cleanup=True):
+    """Start the credis global state store."""
+
+    stdout_file, stderr_file = new_log_files(
+        "credis_master", redirect_output)
+
+    master_port, _ = start_redis_instance(
```

Comment:
There's some code duplication with calling `start_redis_instance` 3 times, maybe it should be in a loop